tag.go

```diff
@@ -12,6 +12,10 @@ type Tag struct {
 	Value string
 }
 
+func T(k, v string) Tag {
```

Comment:
Can we document this function? So we don't trade a govet warning for one in golint ;)